requirements.txt

```diff
@@ -212,12 +206,6 @@ google-resumable-media==2.4.1 \
     # via
     #   google-cloud-bigquery
     #   google-cloud-storage
-# This requirement duplicates the line below, without an extra. We need this to
-# work around https://github.com/pypa/pip/issues/9644 because we can't use
-# --no-deps when this gets installed by GCF
-googleapis-common-protos==1.59.0 \
-    --hash=sha256:4168fcb568a826a52f23510412da405abd93f4d23ba544bb68d943b14ba3cb44 \
-    --hash=sha256:b287dc48449d1d41af0c69f4ea26242b5ae4c3d7249a38b0984c86a4caffff1f
 googleapis-common-protos[grpc]==1.59.0 \
```

Comment:
```suggestion
# This requirement duplicates the line below, without an extra. We need this to
# work around https://github.com/pypa/pip/issues/9644 because we can't use
# --no-deps when this gets installed by GCF
googleapis-common-protos==1.59.0 \
    --hash=sha256:4168fcb568a826a52f23510412da405abd93f4d23ba544bb68d943b14ba3cb44 \
    --hash=sha256:b287dc48449d1d41af0c69f4ea26242b5ae4c3d7249a38b0984c86a4caffff1f
googleapis-common-protos[grpc]==1.59.0 \
```